addons/project/tests/test_project_subtask_ui_flow.py

```diff
@@ -0,0 +1,22 @@
+from odoo import Command
+from odoo.tests import HttpCase, tagged, Form
+
+from .test_project_base import TestProjectCommon
+
+@tagged('-at_install', 'post_install')
+class TestProjectSubtaskUIFlow(HttpCase, TestProjectCommon):
```

Comment:
Why do we need `HttpCase`? 🤔 

Also, I think you can move your test in `test_project_subtasks` python file

Reply:
`HttpCase` was not needed, thanks for pointing that out :)